app/gcp/outputs.tf

```diff
@@ -3,11 +3,6 @@ output "api_service_url" {
   value       = google_cloud_run_v2_service.planqtn_api.uri
 }
 
-output "api_service_account_key" {
-  description = "The service account key for the API service"
```

Comment:
why removing it? This is required by the supabase functions to call into the API...